skactiveml/classifier/_wrapper.py

```diff
@@ -297,25 +359,21 @@ def _fit(self, fit_function, X, y, sample_weight=None, **fit_kwargs):
             ):
                 if fit_function == "partial_fit":
                     fit_kwargs["classes"] = self.classes_
-                    self.estimator_.partial_fit(
-                        X=X_lbld, y=y_lbld_inv, **fit_kwargs
-                    )
+                    self.estimator_.partial_fit(X=X_lbld, **fit_kwargs)
                 elif fit_function == "fit":
-                    self.estimator_.fit(X=X_lbld, y=y_lbld_inv, **fit_kwargs)
+                    self.estimator_.fit(X=X_lbld, **fit_kwargs)
             else:
                 if fit_function == "partial_fit":
                     fit_kwargs["classes"] = self.classes_
```

Comment:
Should we even set classes ourselves? What happens if fit_kwargs already has classes?